docs/contributor/commands-readme.md

```diff
@@ -44,4 +44,5 @@ the default branch.
 The regex in cmd.yml is: `^(\/cmd )([-\/\s\w.=:]+)$` accepts only alphanumeric, space, "-", "/", "=", ":", "." chars.
 
 `/cmd bench --runtime bridge-hub-westend --pallet=pallet_name`
+`/cmd prdoc --pr 5924 --audience runtime_dev runtime_user --bump patch`
```

Comment:
If `pr` is optional, we should not mention it by default. Generally I would expect some more information on what is used as description etc.